contracts/metagov/AaveDelegatorPCVDeposit.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "./SnapshotDelegatorPCVDeposit.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../external/aave/IAaveIncentivesController.sol";
+
+interface IStkAave is IERC20 {
+    function stake(address onBehalfOf, uint256 amount) external;
+    function redeem(address to, uint256 amount) external;
+    function cooldown() external;
+}
+
+/// @title Aave Delegator PCV Deposit
+/// This contract delegates the voting power it holds in the Aave governance.
+/// This contract is designed to be whitelisted as the rewards claimer for the
+/// PCVDeposits that deposit on Aave protocol. This contract will pull rewards
+/// (mostly stkAAVE) directly on this contract, claiming on behalf of other
+/// deposits. Should this contract receive AAVE, they can be staked to stkAAVE
+/// permissionlessly. If this contract is paused, stkAAVE can also be unwrapped
+/// to AAVE after the cooldown period of 10 days. After a cooldown is initiated,
+/// and the 2-day redeem period opens, anyone can unwrap the stkAAVE to AAVE.
+/// @author Fei Protocol
+contract AaveDelegatorPCVDeposit is SnapshotDelegatorPCVDeposit {
+
+    /// @notice the Aave Incentives Controller.
+    IAaveIncentivesController public constant aaveIncentivesController = IAaveIncentivesController(address(0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5));
+
+    /// @notice the AAVE token address.
+    IERC20 public constant aave = IERC20(address(0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9));
+    /// @notice the stkAAVE token address.
+    IStkAave public constant stkaave = IStkAave(address(0x4da27a545c0c5B758a6BA100e3a049001de870f5));
```

Comment:
No need to cast to an address, then cast to an interface. You can just remove the cast to address.